javascript/modules/client/src/internal/interfaces/plugins.ts

```diff
@@ -19,14 +19,13 @@ export interface IClientErc20 extends IClientCore {
     ) => AsyncGenerator<ProposalCreationStepValue>;
     voteProposal: (proposalId: string, vote: VoteValues) => AsyncGenerator<VoteProposalStepValue>;
     executeProposal: (proposalId: string) => AsyncGenerator<ExecuteProposalStepValue>;
-    setPluginConfig: (config: ProposalConfig) => AsyncGenerator<SetPluginConfigStepValue>
-    getMembers: () => Promise<string[]>;
+    getMembers: (addressOrEns: string) => Promise<string[]>;
     getProposal: (propoosalId: string) => Promise<Erc20Proposal>
     getProposals: (params?: IProposalQueryParams) => Promise<Erc20ProposalListItem[]>
   };
   encoding: {
     /** Computes the parameters to be given when creating the DAO, so that the plugin is configured */
-    setPluginConfigAction: (params: ProposalConfig) => DaoAction;
+    setPluginConfigAction: (params: IProposalConfig) => DaoAction;
```

Comment:
Yes we can change the name but then it will also change on the Proposal definition:

```ts
export type Proposal = {
  ...
  settings: IProposalConfig
  ...
}
```
to
```ts
export type Proposal = {
  ...
  settings: IPluginConfig
  ...
}
```

which does not make much sense, before this commit there were 2 interfaces with different name but the same fields, we deleted one of them to avoid duplicates

Reply:
Why should we change Proposal? My point is that one thing is a plugin's settings, and another one is a Proposal's config. Most likely, proposal settings need to respect a plugin's config, otherwise transactions are going to be reverted. But they are not the same.
- If you want to change a plugin's config, you will create a proposal (with its config)
- Such proposal may eventually update the Plugin's internal config